src/Umbraco.Web.UI.Client/src/views/propertyeditors/datepicker/weekdays.prevalues.html

```diff
@@ -0,0 +1,5 @@
+﻿<div ng-controller="Umbraco.PrevalueEditors.WeekdaysController">
+    <div ng-repeat="day in weekdays">
+        <label><input type="checkbox" name="weekdays" checklist-model="selectedDays" checklist-value="day.id" /> {{day.name}}</label>
+    </div>
+</div>
```

Comment:
Maybe the weekdays prevalues doesn't need to be specific datetime picker?
Probably other might want to use this as well? .. I store an array of the selected days, where Sunday=0, Monday=1, etc.